package.json

```diff
@@ -43,83 +44,66 @@
   "dependencies": {
     "array-equal": "^1.0.0",
     "bezier-easing": "^2.0.3",
-    "chroma-js": "^1.1.1",
     "classnames": "^2.2.3",
     "color": "^0.11.3",
     "d3-interpolate": "^1.0.0",
-    "date-arithmetic": "^3.1.0",
-    "lodash.assign": "^4.0.9",
     "lodash.camelcase": "^4.1.0",
```

Comment:
btw, are you aware about lodash's [webpack](https://www.npmjs.com/package/lodash-webpack-plugin) and [babel](https://www.npmjs.com/package/babel-plugin-lodash) plugins?

these can help even further